mlrun/datastore/base.py

```diff
@@ -64,6 +64,17 @@ def is_structured(self):
     def is_unstructured(self):
         return True
 
+    @staticmethod
+    def get_file_url(url):
```

Comment:
please add a link to https://datatracker.ietf.org/doc/html/rfc1808.html#section-2.1 so it will be easy for the next person to understand what you did here